client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -135,13 +134,13 @@ class SiteTools extends Component {
 						isWarning
 					/>
 				}
-				{ showManageConnection &&
+				{ showDisconnectSite &&
```

Comment:
That's the rebase again... I did not remove lines of Manage Connection. There should be an emoji for a facepalm :)